### PR TITLE
Added context information for the datetime module.

### PR DESCRIPTION
**Pull request by mgenereu**
## Description & motivation
I was doing algebraic date math before since I didn't know the timedelta was available. Once I read the dbt module source code, I felt this should be included.

I also updated the example to provide a common use case to get an ISO date for a few days ago that could be used in a SQL context.

### CODE REVIEW

**Comment by mgenereu** [comment]
How do I add a reviewer like @drewbanin ?

File: website/docs/docs/writing-code-in-dbt/jinja-context/modules.md [hunk 1]
```diff
@@ -8,13 +8,17 @@ The `modules` variable in the Jinja context contains useful Python modules for o
 ## datetime
 This variable is a pointer to the Python [datetime](https://docs.python.org/3/library/datetime.html) module.
 
+It includes the modules contexts of `date`, `datetime`, '`time`, `timedelta`, and `tzinfo`.
```
Comment (jtcohen6) [inline website/docs/docs/writing-code-in-dbt/jinja-context/modules.md:11]: Extra single quote:
```suggestion
It includes the modules contexts of `date`, `datetime`, `time`, `timedelta`, and `tzinfo`.
```

**Comment by mgenereu** [comment]
@jtcohen6 How do I complete the merge?

**Comment by clrcrl** [comment]
I can do that! (Someone on our team needs to click the button!)

**Pull request merged**